app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/profile/ProfileFragment.kt

```diff
@@ -0,0 +1,17 @@
+package com.bottlerocketstudios.brarchitecture.ui.profile
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.bottlerocketstudios.brarchitecture.ui.BaseFragment
+import com.bottlerocketstudios.compose.profile.ProfileScreen
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class ProfileFragment : BaseFragment<ProfileViewModel>() {
```

Comment:
@VanyaHuamanBottleRocket Thanks for the refactor to profile! This naming reflects the intent of the UI and class much more clearly :D